plot_pixel_values.py

```diff
@@ -15,48 +17,105 @@ def get_opt():
     parser.add_argument('--pix-filename',
                         default='pixel_values.dat',
                         help='Input pixel values filename')
-
     parser.add_argument('--start',
                         help='Start time (default=2000:001)')
-
-    parser.add_argument('--fit-scaling',
-                        action='store_true',
-                        help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
+                        action='store_true')
     parser.add_argument('--n-brightest',
-                        default=128,
+                        default=64,
                         type=int,
-                        help='Plot the N brightest (either 128 or n**2)')
+                        help='Plot the N brightest (either 64 or n**2)')
 
     args = parser.parse_args()
     return args
 
 opt = get_opt()
 
+T_CCD_REF = -19 # Reference temperature for dark current values in degC
+def dark_scale_model(pars, t_ccd):
+    """
+    dark_t_ref : dark current of a pixel at the reference temperature
+    scale : dark current model scale factor
+    returns : dark_t_ref scaled to the observed temperatures t_ccd
+    """
+    scale, dark_t_ref = pars
+    scaled_dark_t_ref = dark_t_ref * np.exp(np.log(scale) / 4.0 * (T_CCD_REF - t_ccd))
+    return scaled_dark_t_ref
+
+
+def fit_pix_values(t_ccd, esec, id=1):
+    logger = logging.getLogger("sherpa")
+    logger.setLevel(logging.WARN)
+    data_id = id
+    ui.set_method('simplex')
```

Comment:
Do I need to ui.clean() or the like for safety before each of these per-pixel fits?  Or clean on each new loop?